frontend/src/types/Health.ts

```diff
@@ -235,7 +235,7 @@ export const isProxyStatusSynced = (status: ProxyStatus): boolean => {
 };
 
 export const isProxyStatusComponentSynced = (componentStatus: string): boolean => {
-  return componentStatus === 'Synced';
+  return componentStatus === 'Synced' || componentStatus === 'IGNORED';
```

Comment:
```suggestion
  switch (componentStatus.toLowerCase()) {
    case 'ignored':
    case 'synced':
      return true;
    default:
      return false;
  }
```
You can skip this, just a little safer to not rely on case.